packages/server-core/src/media/storageprovider/s3.storage.ts

```diff
@@ -97,13 +127,12 @@ export class S3Provider implements StorageProviderInterface {
     })
   }
 
-  listObjects = async (prefix: string, recursive = false): Promise<StorageListObjectInterface> => {
+  listObjects = async (prefix: string): Promise<StorageListObjectInterface> => {
```

Comment:
we need to keep this recursive option and the delimiter

Reply:
@HexaField  Should we do it like this?

Reply:
yes, though we want to keep the default to false, not true